common/src/main/kotlin/web5/sdk/common/Varint.kt

```diff
@@ -0,0 +1,83 @@
+package web5.sdk.common
+
+/**
+ * Object Varint contains utility functions to encode and decode integers using variable-length quantity (Varint).
+ * Varint is a method of serializing integers using one or more bytes; smaller integers use fewer bytes.
+ * Used by Multicodec (which is used for DidKey)
+ */
+public object Varint {
```

Comment:
Chad + comparision with existing (limited) Varint gives me confidence this will be fine.

Would still be great to have this file and a few others get a big audit in the future

Reply:
tests added